### PR TITLE
Improve the log message in node

### PR DESCRIPTION
**Pull request by jianoaix**
## Why are these changes needed?
Make it more clear what's happening in node based on feedback

## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: node/node.go [hunk 1]
```diff
@@ -163,7 +163,6 @@ func (n *Node) Start(ctx context.Context) error {
 
 	// Build the socket based on the hostname/IP provided in the CLI
 	socket := string(core.MakeOperatorSocket(n.Config.Hostname, n.Config.DispersalPort, n.Config.RetrievalPort))
-	n.Logger.Info("Registering node with socket", "socket", socket)
```
Comment (jianoaix) [inline node/node.go:166]: This is misleading: it will not register if `RegisterNodeAtStart` is false (which is the default).

File: node/node.go [hunk 2]
```diff
@@ -358,6 +358,8 @@ func (n *Node) updateSocketAddress(ctx context.Context, newSocketAddr string) {
 }
 
 func (n *Node) checkRegisteredNodeIpOnChain(ctx context.Context) {
+	n.Logger.Info("Start checkRegisteredNodeIpOnChain goroutine in background to subscribe the operator socket change events onchain")
```
Comment (jianoaix) [inline node/node.go:361]: Here and below: the start of an important routine

File: node/node.go [hunk 3]
```diff
@@ -207,12 +207,12 @@ func (n *Node) expireLoop() {
 		// least have 1 second.
 		timeLimitSec := uint64(math.Max(float64(n.Config.ExpirationPollIntervalSec)*gcPercentageTime, 1.0))
 		numBatchesDeleted, err := n.Store.DeleteExpiredEntries(time.Now().Unix(), timeLimitSec)
-		n.Logger.Info("GC cycle deleted", "num batches", numBatchesDeleted)
+		n.Logger.Info("Complete an expiration cycle to remove expired batches", "num expired batches found and removed", numBatchesDeleted)
```
Comment (jianoaix) [inline node/node.go:210]: Expand what it's doing so it's clear for users reading the log.

**Review by ian-shim — APPROVED** [review]

**Comment by wesfloyd** [comment]
Hi @jianoaix - sharing two requests for our log message improvements:
1) Can we enhance the log message to be more clear for when the Operator is ejected from the active operator set? Per [these docs here](Add a log message to indicate the user has not yet opted-in with EigenDA prior to running the service) the current log message is: "GC cycle deleted  "
2) Can we add a log message to indicate the user has not yet opted-in with EigenDA prior to running the service? Per [this support ticket](https://app.hubspot.com/contacts/24384477/record/0-5/2093047666).

**Comment by jianoaix** [comment]
Thanks for feedback, these are good points.
1: The ejection isn't happening at DA Node, so will need to address separately.
2: Noted, will add that log (but will not include in v0.1.1 since it's not as high priority)

**Pull request merged**